Explorer/Assets/DCL/PluginSystem/Global/ExplorePanelPlugin.cs

```diff
@@ -230,7 +230,7 @@ public class ExplorePanelSettings : IDCLPluginSettings
             public AssetReferenceGameObject ExplorePanelPrefab;
 
             [field: SerializeField]
-            public BackpackSettings BackpackSettings { get; private set; }
+            public BackpackSettings? BackpackSettings { get; private set; }

```

Comment:
If you added this to avoid the warning, consider adding a `!` if we're sure it isn't null on line `214`

Reply:
tbh, I probably added it by accident when letting Rider do some auto-refactor thing...